File: app/src/main/java/org/mifos/mobile/ui/views/NotificationView.java
package org.mifos.mobile.ui.views;

import org.mifos.mobile.models.notification.MifosNotification;
import org.mifos.mobile.ui.views.base.MVPView;

import java.util.List;

/**
 * Created by dilpreet on 14/9/17.
 */

public interface NotificationView extends MVPView {

    void showNotifications(List<MifosNotification> notifications);

    void showError(String msg);
}

File: app/src/main/java/org/mifos/mobile/ui/views/NotificationView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.models.notification.MifosNotification
import org.mifos.mobile.ui.views.base.MVPView

/**
 * Created by dilpreet on 14/9/17.
 */
interface NotificationView : MVPView {
    fun showNotifications(notifications: List<MifosNotification?>?)
    fun showError(msg: String?)
}